joomla/joomla-cms
# [4.0] Rewrite sampledata-process.js

Pull Request for Issue #19263

### Summary of Changes

- Rewrite in vanilla JS
- Use Custom Element alerts instead of BS
- Progress bar now increments as per the step it's on.
- Create minified file

### Testing Instructions

If you want to test, you'll need to ensure you have no sample data installed already. So install a fresh copy of Joomla and attempt to install sample data.

If you **do** have sample data already installed, the process should stop on step 1 and display an error alert.
